config/prometheus.template.yml

```diff
@@ -9,6 +9,10 @@ scrape_configs:
     static_configs:
       - targets: ['aws_metrics:80']
 
+  - job_name: 'bb8'
+    static_configs:
+      - targets: ['annex.montagu.dide.ic.ac.uk:5001']
```

Comment:
Can we take the opportunity to add this to the arch diagram and the Annex.md file in docs on montagu-machine? Or even just raise a ticket so we remember to do this!